### PR TITLE
Fix backup/README.md dead link

### PR DESCRIPTION
**Pull request by LeoSpyke**
Fix a dead link to "info about backup solutions"

### CODE REVIEW

**Comment by chrisguida** [comment]
Needs rebase

File: backup/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This plugin will maintain clean database backups to another location. It uses
 the `db_write` hook to make sure to always have a backup that is not missing any
 state updates and is not potentially harmful.
 
-Related info about backup solutions: https://github.com/ElementsProject/lightning/blob/master/doc/BACKUP.md
+Related info about backup solutions: https://github.com/ElementsProject/lightning/blob/master/doc/beginners-guide/backup-and-recovery.md
```
Comment (chrisguida) [inline backup/README.md:9]: This link appears to be broken as well

**Review by chrisguida — DISMISSED** [review]

**Comment by chrisguida** [comment]
Can you clean up the commit history here? For some reason there are 5 commits, there needs to be just 1 commit with your change. I tried to merge master into your branch but it did not fix the problem, sorry.

**Review by chrisguida — CHANGES_REQUESTED** [review]
needs history cleanup

**Comment by cdecker** [comment]
Contributor does not allow maintainers to amend the PR. As such we need them to rebase in order to be merged. If the author does not, I will just cherry-pick the one useful commit onto a fixup PR later.

**Pull request closed without merging**